code_of_conduct.md

```diff
@@ -0,0 +1,130 @@
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
```

Comment:
I think this may almost be better and more generically inclusive if just left at "We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone", given the intent of the paragraph.

Reply:
We want to keep the standard contributor covenant code of conduct, as we didn't write and it provides more of a standard.

Reply:
I personally like being explicit about this. Are there examples that you can think of which is not included?